tests/layer_tests/pytorch_tests/test_div.py

```diff
@@ -54,7 +52,38 @@ def forward(self, input_tensor, other_tensor):
     ]))
 
     @pytest.mark.nightly
-    def test_div(self, input_array, input_type, other_array, other_type, rounding_mode, ie_device, precision, ir_version):
+    def test_div_float(self, input_array, input_type, other_array, other_type, rounding_mode, ie_device, precision, ir_version):
+        self.input_array = input_array
+        self.input_type = input_type
+        self.other_array = other_array
+        self.other_type = other_type
+        self._test(*self.create_model(rounding_mode), ie_device, precision, ir_version)
+
+
+    @pytest.mark.parametrize(("input_array", "other_array"), [
+        [np.random.rand(5, 5), np.random.rand(1)],
```

Comment:
You also need to restrict division by zero

Reply:
@mvafin Done, changed to `np.random.uniform` with `low=0.1`.